handbook/engineering/observability/cloudflare.md

```diff
@@ -0,0 +1,37 @@
+# Sourcegraph Cloudflare monitoring developer guide
```

Comment:
Can you add a brief description and link to this in the [Sourcegraph Cloud section in the architecture](https://about.sourcegraph.com/handbook/engineering/observability/monitoring_architecture#sourcegraph-cloud)?

Reply:
Added